### PR TITLE
#164069303 Setup and deploy base application to heroku 

### PR DESCRIPTION
**Pull request by walimike**
#### What does this PR do?
- Enable users access the base application. 
- Create pipelines for review apps on heroku 

#### Description of tasks to be completed?
 - Add Procfile to root of app
 - Add app.json file for pipelines
 - Update requirements.txt for heroku build

#### How should this be manually tested?
When user navigates to base URL found [here](https://ah-backend-groot.herokuapp.com/).
Then user should access the live application

#### Any background context you want to provide?
Pipelines have three stages, the review apps for PRS, the staging which has the develop branch and the production.

#### What are the relevant pivotal tracker stories?
#164069303

##### Pipelines image
screenshot 2019-02-27 at 10 37 31

##### Base application
screenshot 2019-02-27 at 13 05 10

### CODE REVIEW

**Comment by akram256** [comment]
So can we access the pipelines

**Comment by walimike** [comment]
Yes @akram256 it is provided on the slack channel

File: authors/urls.py [hunk 1]
```diff
@@ -25,6 +25,6 @@
 
     path('api/', include('authors.apps.authentication.urls', )),
     path('swagger/', schema_view),
-    path('docs/', coreapi_docs),
+    path('', coreapi_docs),
```
Comment (PatrickCmd) [inline authors/urls.py]: @walimike Please make this url just to redirect to the coreapi docs, so you should maintain both the root url and docs url.
See the rest framework documentation on how to do redirects.

**Comment by PatrickCmd** [comment]
@walimike app.json should be at the root of the project. If you take a look at the review apps in our heroku pipelines are failing because of this. Your review app has to successfully be created and tested before this PR can be merged.

**Review by PatrickCmd — APPROVED** [review]

**Pull request merged**